### ISSUE TITLE
Exception occured during first run

### ISSUE THREAD

**Issue by fullstackdevsalmoh** [mistrypragnesh40/CreatingWebAPIDemo#1]
I added new migration then I executed update-database, but it gives the following exception: 

| $exception | {"The requested address is not valid in its context."} | System.Net.Sockets.SocketException

Can you point me to correct this please?

**Comment by nagyed** [comment]
I adjusted connection string but when i run the API it gives error as follows : An unhandled exception of type 'System.Net.Sockets.SocketException' occurred in System.Private.CoreLib.dll
The requested address is not valid in its context. this exception stopped app at line no. 96 in program.cs which is "app.Run();" how can I solve this?

**Comment by CentauriDK** [comment]
I solved it by making some changes in the file /Properties/launchSettings.json
change both lines containing applicationUrl from the "192.168.0.185" to "localhost"
Then it worked for me.

**Comment by CentauriDK** [comment]
Her is my compleate config, because if you wante to use SSL and be able to use that with the MAUI Example your need to use SSL and change the BaseUrl ind the Models/settings.cs in the LoginFlowInMauiBlazorApp if uou are using that.

{
  "$schema": "https://json.schemastore.org/launchsettings.json",
  "iisSettings": {
    "windowsAuthentication": false,
    "anonymousAuthentication": true,
    "iisExpress": {
      "applicationUrl": "http://localhost:8188",
      "sslPort": 44318
    }
  },
  "profiles": {
    "CRUDOperationUsingWEBAPI": {
      "commandName": "Project",
      "launchBrowser": true,
      "launchUrl": "swagger",
      "environmentVariables": {
        "ASPNETCORE_ENVIRONMENT": "Development"
      },
      "applicationUrl": "https://localhost:44318",
      "dotnetRunMessages": true
    },
    "IIS Express": {
      "commandName": "IISExpress",
      "launchBrowser": true,
      "launchUrl": "swagger",
      "environmentVariables": {
        "ASPNETCORE_ENVIRONMENT": "Development"
      }
    }
  }
}

**Comment by mistrypragnesh40** [comment]
Yes, @CentauriDK , Right now that project pointing to my system ip configuration. You might need to change some setting in launchsetting.json & might need to set TrustServerCertificate=True; in connection string.

IN AppSetting.json 
"ConnectionStrings": { "StudentDB": "Server=.\\;Database=StudentDB;Trusted_Connection=True;TrustServerCertificate=True;" },
